app/src/lib/ui/OrientationRequest/ProOrientationRequestBanner.svelte

```diff
@@ -21,6 +21,11 @@
 		reorientationRequest.status == 'denied' || reorientationRequest.status == 'accepted'
 			? formatDateLocale(reorientationRequest.decidedAt)
 			: formatDateLocale(reorientationRequest.createdAt);
+
+	$: [orientationLabel, displayedOrientation] =
+		decision == 'acceptée'
```

Comment:
```suggestion
		reorientationRequest.status == 'accepted'
```
Peut-être vaut-il mieux rester cohérent avec les tests fait au dessus au lieu de tester la valeur "d'affichage" ?